src/deps/libuv.zig

```diff
@@ -2848,6 +2848,10 @@ fn StreamMixin(comptime Type: type) type {
             }
             return .{ .result = {} };
         }
+        /// Be aware that this only works with pipes on Windows.
+        pub fn setBlocking(this: *Type, blocking: bool) void {
+            _ = uv_stream_set_blocking(@ptrCast(this), @intFromBool(blocking));
```

Comment:
Is this still necessary?

Reply:
cc @cirospaciari

Reply:
not any more, sorry I should have removed this when changed the implementation

Reply:
roger